config/locales/en.yml

```diff
@@ -2,13 +2,27 @@ en:
   activerecord:
     errors:
       models:
+        organisation:
+          attributes:
+            type:
+              blank: Select an organisation type
         provider:
           attributes:
             provider_code:
               taken: Provider already exists
+              blank: Provider name can't be blank
+        school:
+          attributes: 
+            urn: 
+              taken: School already exists
```

Comment:
Don't know if this is actually a requirement but when you add a school that is already there you have an error next to the field as well as at the top of the page, I think you're missing the error next to the field. 

This is from the prototype
https://itt-mentoring-beta-94341ca35a2c.herokuapp.com/support/organisations/new/school
![Screenshot from 2024-01-04 15-21-18](https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/0a16b9dc-a29a-456a-b774-6f92d0b9a5ac)

Reply:
Resolved in [Improve school form error handling](https://github.com/DFE-Digital/itt-mentor-services/pull/73/commits/ba6519cdcd2a9e194a2109f8b2b5ac1f233cb343)